spec/onc_certification_g10_test_kit/bulk_data_group_export_validation_spec.rb

```diff
@@ -150,6 +172,23 @@ def run(runnable, inputs = {})
 
       expect(result.result).to eq('pass')
     end
+
+    it 'validates all lines and saves errors for the first failed line' do
+      stub_request(:get, endpoint.to_s)
+        .with(headers: { 'Accept' => 'application/fhir+ndjson' })
+        .to_return(status: 200, body: contents, headers: headers)
+
+      stub_request(:post, "#{validator_url}/validate?profile=http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient")
```

Comment:
We expect this request to happen twice, so as a sanity check we should make sure that actually happens.

```
validation_request = stub_request(...)

...the rest of the spec...

expect(validation_request).to have_been_made.twice
```